app-text/tesseract_data/tesseract_data-3.04.00.recipe

```diff
@@ -0,0 +1,158 @@
+SUMMARY="Languages data files for Tesseract OCR engine"
+DESCRIPTION="Tesseract OCR can be fully trained to recognize new language and script. \
+A set of files for already, community made, trained languages are available, splitted \
+in seperate package per language."
+HOMEPAGE="http://code.google.com/p/tesseract-ocr"
```

Comment:
Some typos:
"Tesseract OCR can be fully trained to recognize new _languages_ and _scripts_. \
 A set of files for [_already, --delete_]community made, trained languages are available as \
 _separate packages per language._"